test/OPs_inside_cubics_3_variable.jl

```diff
@@ -530,3 +813,146 @@ function nki2ind(n,k,i)
 end
 
 deg(k,i) = Int64(3*floor(k/2)) + mod(k,2) + 1-i
```

Comment:
```suggestion
deg(k,i) = 3*(k ÷ 2) + mod(k,2) + 1-i
```

Reply:
@MarcoFasondini I'll leave you to commit the suggestions, just thought it would be good to see the proposed changes so you pick up some Julia tricks